app/Notifications/CommentNotification.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Comment;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class CommentNotification extends Notification
+{
+    use Queueable;
+
+    private Comment $comment;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(Comment $comment)
+    {
+        $this->comment = $comment;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param mixed $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param mixed $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->line('There are new comments on a booking.')
+            ->action('View Review', url('/bookings/' . $this->comment->booking_id . '/review'))
+            ->line('Thank you for using our application!');
+    }
+
+    /**
+     * Get the array representation of the notification.
+     *
+     * @param mixed $notifiable
+     * @return array
+     */
+    public function toArray($notifiable)
+    {
+        return [
+            //
+        ];
+    }
```

Comment:
This should contain the "object" form of the notification. 
```suggestion
    public function toArray($notifiable)
    {
        return [
            'message' => "" //Add whatever message 
            'link' => url('/bookings/' . $this->comment->booking_id . '/review')
            // Add more context info here as requested in my other review 
        ];
    }
```

Reply:
as we discussed, not needed anymore